src/QGONexum.py

```diff
@@ -88,6 +88,34 @@
 #from qiskit_optimization.algorithms import CplexOptimizer
 
 
+
+class QAOANode:
+
+    neighbors = []
+
+    def __init__(self, point: gpd.GeoSeries, name: str):
+        self.point = point
+        self.coordinates: tuple[float, float] = ()
+        try:
+            self.coordinates = (point["geometry"].x, point["geometry"].y)
+        except: 
+            self.coordinates = (point["geometry"].centroid.x, point["geometry"].centroid.y)
+            
+        self.name = name
+    
+    def setCoordinates(self, coordinates: tuple[float, float]):
+        self.coordinates = coordinates
+
+    def getCoordinates(self) -> tuple[float, float]:
+        return self.coordinates
+    
+    def changeName(self, new_name: str):
+        self.name = new_name
+        
+    def __str__(self):
+        return self.name
```

Comment:
Moved this class up so type hints could be added to `getEdgesAndCapacities` below.